cdap-unit-test/src/test/java/co/cask/cdap/spark/SparkTestRun.java

```diff
@@ -411,6 +412,13 @@ public void testSparkWithLocalFiles() throws Exception {
                             SparkAppUsingLocalFiles.ScalaSparkUsingLocalFiles.class.getSimpleName(), "scala");
   }
 
+  @Test
+  public void testSparkWithSecureStore() throws Exception {
+    ApplicationManager applicationManager = deployApplication(SparkSecureStoreApp.class);
```

Comment:
Can you merge the app and the test with the `testSecureStoreRuntimeApis`? It will cut down the unit-test runtime.